app/models/manageiq/providers/azure/cloud_manager/provision/configuration.rb

```diff
@@ -0,0 +1,6 @@
+module ManageIQ::Providers::Azure::CloudManager::Provision::Configuration
+  def userdata_payload
+    return unless raw_script = super
```

Comment:
@bdunne -This is a bit tough to read - but after understanding the context - it makes sense.